modules/ddb_cover_upload/js/ddb_cover_upload_dialog.admin.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Use Drupal's built in modal.
+ */
+
+(function ($) {
+  Drupal.behaviors.ddbCoverUpload = {
+    attach: function (context, settings) {
+      let dialogLink = '.opener-modal';
+      $(dialogLink).click(function() {
+        let dialogId = '#' + $(this).attr("data-dialog");
```

Comment:
I see you use `let` and `var` interchangeably. Is there any reason for using `let` in some places, and, if not, would it not be safer to just use `var` everywhere?

Reply:
Fixed :-)